Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -109,3 +110,12 @@ Make sure your Android paths are properly configured. If you continue to encount
 **Edit > Preferences > External Tools > Android**
 
 ![AndroidToolsConfig](../Images/CrossPlatform/OculusQuest/AndroidToolsConfig.png)
+
+### The ovrCameraRigPrefab of OculusXRSDKDeviceManagerProfile has not been assigned.
+
+If you encounter an error that looks like this
+
+**UnassignedReferenceException: The variable ovrCameraRigPrefab of OculusXRSDKDeviceManagerProfile has not been assigned. You probably need to assign the ovrCameraRigPrefab variable of the OculusXRSDKDeviceManagerProfile script in the inspector.**
+
+Then the project is not fully integrated with the Oculus Unity Modules. Navigate to Mixed Reality Toolkit > Utilities > Oculus > Integrate Oculus Integration Unity Modules and run it. 
+Doing so populate the **Prefab reference** fields of the **OculusXRSDKDeviceManagerProfile**.
```

Comment:
This is a case I hadn't realized. Does the `Integrate Oculus Integration Unity Modules` _only_ update the profile that ships with MRTK? What if I clone it before running the menu item so I can customize it?

Reply:
Would love to know if that's the case - having custom profiles is a core part of MRTK's workflow

Reply:
It'll only hit the first instance of AssetDatabase.FindAssets("t:OculusXRSDKDeviceManagerProfile"); at the moment, I didn't consider people making their own OculusXRSDKDeviceManagerProfile's when writing this up :(

Reply:
tracking issue here https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8767